src/clusterfuzz/_internal/system/process_handler.py

```diff
@@ -649,3 +649,19 @@ def terminate_processes_matching_cmd_line(match_strings,
     if any(x in process_path for x in match_strings):
       if not any([x in process_path for x in exclude_strings]):
         terminate_process(process_info['pid'], kill)
+
+
+def processes_are_healthy(expected_processes):
```

Comment:
Sorry I meant: 

We can add a helper to `process_handler` that checks if a set of processes are running.

e.g.

```python
def processes_are_running(expected_processes):
 ...
```

and we can call that in `health_check_responder.py`.